mlflow/server/js/src/model-registry/components/CompareModelVersionsView.js

```diff
@@ -273,6 +276,22 @@ export class CompareModelVersionsViewImpl extends Component {
           >
             <ParallelCoordinatesPlotPanel runUuids={runUuids} />
           </TabPane>
+          <TabPane
+            tab={
+              <FormattedMessage
+                defaultMessage='Box Plot'
+                description='Tab pane title for box plot on the compare runs page'
+              />
+            }
+            key='4'
+          >
+            <CompareRunBox
```

Comment:
Can we move this after `CompareRunContour`?

Reply:
Yeah sure